python/graphstorm/gconstruct/file_io.py

```diff
@@ -30,6 +30,64 @@
 
 from .utils import HDF5Handle, HDF5Array
 
+
+def read_index(data_file, column=None):
+    """ Read the index from a JSON/parquet file.
+
+    Each row is a JSON object that contains an index.
+
+    Parameters
+    ----------
+    data_file : str
+        The file that contains the index.
+    column : str
+        Column name on parquet which contains the index
+
+    Returns
+    -------
+    Numpy array : the index array.
+    """
+    # Extract the extension from the filename
+    _, extension = os.path.splitext(data_file)
+
+    # Normalize the extension to ensure case insensitivity
+    extension = extension.lower()
+
+    if extension == '.json':
```

Comment:
Please add `json files should be ended with .json and parquet files should be ended with parquet` in the doc.